### PR TITLE
add some missing parts in about section

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -38,12 +40,19 @@ test:
...
 
 about:
-  home: http://sourceforge.net/projects/p7zip/
-  summary: p7zip is a quick port of 7z.exe and 7za.exe (command line version of 7zip, see www.7-zip.org ) for Unix.
+  home: https://sourceforge.net/projects/p7zip/
+  summary: p7zip is a port of the Windows programs 7z.exe and 7za.exe provided by 7-zip.
+  description: |
+    7-zip is a file archiver with the highest compression ratio. See
+    for more information www.7-zip.org site.
   license: LGPL-2.0-or-later
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: It has the `unRAR` restriction:
```
  Licenses for files are:

    1) CPP/7zip/Compress/Rar* files:  GNU LGPL + unRAR restriction
    2) All other files:  GNU LGPL

  The GNU LGPL + unRAR restriction means that you must follow both 
  GNU LGPL rules and unRAR restriction rules.
```
But it's not spdx compliant.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ package:
   version: {{ version }}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:7]: The build number should be `0`.

**Review by sumit0190 — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (katietz) [reply recipe/meta.yaml]: should be address

**Review by pseudoyim — APPROVED** [review]
License name for RAR components found here: https://scancode-licensedb.aboutcode.org/rar-winrar-eula.html

File: recipe/meta.yaml (hunk 2 above)
Comment (katietz) [reply recipe/meta.yaml:7]: done, thanks.  it is something from CF about bioconda's version

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**